src/org/opendatakit/briefcase/reused/UncheckedFiles.java

```diff
@@ -301,6 +301,16 @@ public static InputStream newInputStream(Path path, OpenOption... options) {
     }
   }
 
+  public static void closeInputStream(InputStream inputStream) {
+    if (inputStream == null)
+      return;
```

Comment:
This is breaking the existing conventions about allowing nulls.

If we ever receive `null` inputstream we should throw an error here. To avoid them, we should change the calling site, not these utility methods, which operate under narrow set of constraints.